cfpb/sbl-frontend
# [Institution profile pages] Crumb trail spacing

Closes #337

## Changes

- CrumbTrail - Support independent rendering
- [[FormWrapper] Support removing top margin](https://github.com/cfpb/sbl-frontend/commit/af0b2d17c38a959c5133d237d2e4cca16663f05b)
- [[Page][View institution] Refactor to render correct CrumbTrail spacing](https://github.com/cfpb/sbl-frontend/commit/f318e221aacc1c31aeb04311c358784fa69875aa)
- [[Page][Update institution] Refactor to render correct CrumbTrail spacing](https://github.com/cfpb/sbl-frontend/commit/d9caa2bb00c1bcea19a70c1cc58c4861ebd7c8dc)
- [Page][Summary] Update CrumbTrail spacing

## How to test this PR

1. Visit http://localhost:8899/institution/123456789TESTBANK123/update
2. Verify CrumbTrail is 30px from site header
3. Visit http://localhost:8899/institution/123456789TESTBANK123
4. Verify CrumbTrail is 30px from site header

## Screenshots
View institution profile
<img width="1055" alt="Screenshot 2024-03-28 at 5 31 29 PM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/a6f260b7-84ac-4fee-9414-d584e0cd5488">

Update institution profile

<img width="1061" alt="Screenshot 2024-03-28 at 5 31 16 PM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/c42c7a4a-d1e6-4723-bd52-883738121b37">

Summary

<img width="1372" alt="Screenshot 2024-04-01 at 10 11 49 AM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/ab7245f7-576c-4934-9e8d-c401d444143c">
